packages/destination-actions/src/destinations/clevertap/index.ts

```diff
@@ -0,0 +1,48 @@
+import type { DestinationDefinition } from '@segment/actions-core'
+import type { Settings } from './generated-types'
+
+import userUpload from './userUpload'
+
+const destination: DestinationDefinition<Settings> = {
+  name: 'Clevertap( Actions)',
+  slug: 'clevertap',
+  mode: 'cloud',
+
+  authentication: {
+    scheme: 'custom',
+    fields: {
+      clevertapAccountId: {
+        label: 'X-CleverTap-Account-Id: ACCOUNT_ID',
+        description: 'Your CleverTap Account ID.',
+        type: 'string',
+        required: true
+      },
+      clevertapPasscode: {
+        label: 'X-CleverTap-Passcode: PASSCODE',
```

Comment:
Same here re: human-friendly

```suggestion
        label: 'CleverTap Passcode',
```